src/components/Editor.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react"
+import ReactMde from "react-mde"
+import Showdown from "showdown"
+import "react-mde/lib/styles/css/react-mde-all.css";
+
+
+export default function Editor({ currentNote, updateNote }) {
+    const [selectedTab, setSelectedTab] = useState("write")
+
+    const converter = new Showdown.Converter({
+        tables: true,
+        simplifiedAutoLink: true,
+        strikethrough: true,
+        tasklists: true,
+    })  
+
+    return (
+        <section className="pane editor">
+            <ReactMde
+                value={currentNote.body}
+                onChange={updateNote}
+                selectedTab={selectedTab}
+                onTabChange={setSelectedTab}
+                generateMarkdownPreview={(markdown) =>
+                    Promise.resolve(converter.makeHtml(markdown))
```

Comment:
you can improve by using a modern way of creating promises

Reply:
I think this way code is shorter than using async/await

Reply:
it's not only about writing a few lines code but also using modern syntax that makes asynchronous code more easier to read which can also be achieved in a few lines of code, I recommend you use a modern way of creating promises.

Reply:
Yes, You're right!

Reply:
Let me work on that